crmsh/sbd.py

```diff
@@ -241,6 +250,28 @@ def _enable_sbd_service(self):
             # in init process
             bootstrap.invoke("systemctl enable sbd.service")
 
+    def _adjust_systemd(self):
+        """
+        Adjust start timeout for sbd when set SBD_DELAY_START
+        """
+        if not self._delay_start:
+            return
+
+        # TimeoutStartUSec default is 1min 30s, need to parse as seconds
+        cmd = "systemctl show service -p TimeoutStartUSec|\
```

Comment:
This looks super anti-python.
Why don't you open the file and apply python regular expressions. Using `grep, sed, awk` looks horrible

Reply:
Changed, thanks!